### DISCUSSION TITLE
Readme images does not loading

### DISCUSSION THREAD

**Discussion by athulpython** [community/community#41911] [Repositories]
### Select Topic Area

Bug

### Body

readme added images doesn't loading  why

**Comment by mohamedzhioua** [comment]
Just had the same experience.  A different browser (edge, ff etc) works fine.  Even chrome guest browsing worked.

**Comment by jfullstackdev** [comment]
that happens from time to time, particularly SVGs, the second time I reload it, it will load successfully

**Comment by krshrimali** [comment]
FWIW, this seems to be an issue with Jio Internet in India (I'm not sure about this so others can verify, cc: @MOHITWANKHADE who also verified). Someone mentioned that changing the proxy worked, and some are using VPN to temporarily resolve this. I'm not sure but sharing this information nonetheless. \

**Reply by athulpython** [reply]
yes your information is correct  jio internet issue
